### PR TITLE
Added GitHub Sponsors Review Template

### LINKED ISSUE #4991: [FEATURE REQUEST] github sponsors reviews
Issue templates go in this folder `.github/ISSUE_TEMPLATE`  https://github.com/EddieHubCommunity/support/tree/main/.github/ISSUE_TEMPLATE

You can use this issue template as an example https://github.com/EddieHubCommunity/support/blob/main/.github/ISSUE_TEMPLATE/opensource-review.yml

Some suggestions...

```yaml
name: GitHub Sponsors Profile Review
description: Please review my GitHub Sponsors profile on a livestream
title: "[REVIEW] <profile name here>"
labels: [gh sponsor review, live stream review]
body:
  - type: input
    id: link
    attributes:
      label: GitHub Sponsor profile link
    validations:
      required: true
  - type: markdown
    attributes:
      value: |
        You can also join our Discord community [here](http://discord.eddiehub.org)
        Feel free to check out other cool repositories of the EddieHub Community [here](https://github.com/EddieHubCommunity)

### PR DESCRIPTION
**Pull request by krupalitrivedi**
closes #4991 

#### What does this PR do?
The Pull Request adds the file for the GitHub Sponsors Profile Review Template!

### CODE REVIEW

**Review by eddiejaoude — DISMISSED** [review]
Thank you 👍

File: .github/ISSUE_TEMPLATE/github-sponsors-profile-review.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: GitHub Sponsors Profile Review
+description: Please review my GitHub Sponsors profile on a livestream
+title: "[REVIEW] <profile name here>"
+labels: [gh sponsor review, live stream review]
```
Comment (Panquesito7) [inline .github/ISSUE_TEMPLATE/github-sponsors-profile-review.yml:4]: Note: a new label should be created for this. :)

File: .github/ISSUE_TEMPLATE/github-sponsors-profile-review.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    id: link
+    attributes:
+      label: GitHub Sponsor profile link
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        You can also join our Discord community [here](http://discord.eddiehub.org)
+        Feel free to check out other cool repositories of the EddieHub Community [here](https://github.com/EddieHubCommunity)
```
Comment (Panquesito7) [inline .github/ISSUE_TEMPLATE/github-sponsors-profile-review.yml:16]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
        Feel free to check out other cool repositories of the EddieHub Community [here](https://github.com/EddieHubCommunity)

```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

**Review by Panquesito7 — COMMENTED** [review]
These could be done later after merging the PR if wanted. 🙂

File: .github/ISSUE_TEMPLATE/github-sponsors-profile-review.yml (hunk 1 above)
Comment (naomi-lgbt) [reply .github/ISSUE_TEMPLATE/github-sponsors-profile-review.yml:4]: image

Label is present. (:

Comment (eddiejaoude) [reply .github/ISSUE_TEMPLATE/github-sponsors-profile-review.yml:4]: I remembered this time to create it first 👍  - thank you for checking

File: .github/ISSUE_TEMPLATE/github-sponsors-profile-review.yml (hunk 2 above)
Comment (eddiejaoude) [reply .github/ISSUE_TEMPLATE/github-sponsors-profile-review.yml:16]: Good spot 👍

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**